fluidefi/fluidefi-caspernet-aggregator-tools
# Block hours fetcher

**Summary of change:**
- Added a new micro-service, block hours fetcher, which returns the latest block hour. 

**List of tasks:** 

- [x] create entity.
- [x] create query.

**Steps for testing**
- Make sure your local git is up to date
```bash
git pull
```

- Checkout the branch `feature/block-hours-fetcher`
```bash
git checkout feature/block-hours-fetcher
```

- Run the test across the test-net
```bash
npm run test
```

- Confirm block-finder test has passed.
- Run example.
